src/Glpi/OAuth/Server.php

```diff
@@ -257,14 +257,14 @@ private static function doGenerateKeys(): void
         try {
             $key = openssl_pkey_new($config);
         } catch (OpensslException $e) {
-            throw new RuntimeException("Unable to generate keys: " . $e->getMessage(), $e->getCode(), $e);
+            throw new RuntimeException("Unable to generate keys: " . $e->getMessage(), 0, previous: $e);
```

Comment:
Small note but is there a reason to manually set the code to 0 here ? Since you use named parameter you could avoid setting the code if you don't need to.

Reply:
It make sense to set it from the previous exception, why shouldn't we ?